docs/howto/account-billing/delete-account.md

```diff
@@ -17,6 +17,20 @@ Choose a reason for leaving and type in your current password. Click the red but
 
 ![Picture where you choose a option why you leave us. A field where you type your current password with a red delete button.](assets/reasonforleaving.png)
 
+
+## Using the {{legal_docs.termination_form.name}}
+
+You may be unable to use the standard account deletion procedure using the {{gui}}, because you have lost your administrative credentials or you cannot access your account owner's email address.
```

Comment:
Since there is a repetition of "use", and since I always find it somewhat strange to say (or even think) "use the procedure", I would suggest a slight re-phrase here.

```suggestion
You may be unable to follow the standard account deletion process using the {{gui}}, because you have lost your administrative credentials or you cannot access your account owner's email address.
```

Reply:
Sounds good.